docs-kits/kits/Digital Twin Kit/page_adoption-view.md

```diff
@@ -0,0 +1,73 @@
+---
+id: Adoption View Digital Twin Kit
+title: Adoption View
+description: 'Digital Twin Kit'
+sidebar_position: 2
+---
+
+### Digital Twin Kit
+<!--
+Adoption View of the Kit.
+-->
+
+<!-- !Mandatory! -->
+## Vision & Mission
+
+### Vision
+
+The aim of the Digital Twin KIT is to trace parts and materials across the entire value chain to enable data driven use 
+cases over all n-tier levels without compromising data sovereignty. This KIT enables data and app providers to deliver 
+solutions for building data chains and to send quality notifications on all levels and industries.
+
+To provide the Catena-X Automotive Network with a uniform infrastructure to enable data-level interoperability between
+Business Partners is the purpose of the Digital Twin Kit. Regardless of the data's provenance, the KIT sets the scene
+for a comprehensive landscape of distributed Digital Twins of assets (mostly parts) along the entire lifecycle of the
+supply chain.
+
+### Mission
+
+The Digital Twin KIT bundles the necessary standards, APIs, interaction patterns and reference implementations on how
+to build a Digital Twin for an asset. By adopting established standards and integrating them, the Digital Twin Kit leverages
+the conceptual and implementation work of existing communities while contributing feedback and FOSS back to the public.
+A priority is the seamless integration with other Catena-X developments such as the Connector KIT. 
+
+### Customer Journey
+
+With the Digital Twin KIT, all roles mentioned in the Operating Model Whitepaper are given the necessary tooling to format
+their data and APIs in a standardized manner.
+
+<!-- !Mandatory! -->
+## Business Value
+
+Point-to-Point integration between Business Partners is hard. On the one hand, all questions of sovereignty, authorization, 
+authentication must be agreed upon and implemented. That is covered by the Connector-KIT and the services it relies upon.
+What this KIT adds is a set of technologies that reduce the integration efforts on the data level. Data Consumers can
```

Comment:
Why capital letters for "Data Consumers"?

Reply:
see above